chart/kubeapps/values.yaml

```diff
@@ -542,19 +542,22 @@ dashboard:
 ## ref: https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml
 ##
 postgresql:
+  # Enable PostgreSQL. This should be true
+  enabled: true
   ## Enable replication for high availability
   replication:
     enabled: true
   ## Create a database for Kubeapps on the first run
   postgresqlDatabase: assets
+  ## PostgreSQL admin password (used when `postgresqlUsername` is not `postgres`)
```

Comment:
Confused - `postgresqlPassword` is an admin password iff `postgresqlUsername` *is* `postgres` . It is only when `postgresqlUsername` is not `postgres` that `postgresqlPassword` will be the non-admin password and `postgresqlPostgresqlPassword` is used for the admin password. So I'm confused by the above comment, even though I assume you grabbed it from other charts?

Reply:
right, no, my bad, I copied the comment from a `postgresqlPostgresqlPassword`